services/src/main/java/org/keycloak/protocol/oidc/OIDCLoginProtocolService.java

```diff
@@ -215,8 +213,9 @@ public Object issueUserInfo() {
     /* old deprecated logout endpoint needs to be removed in the future
     * https://issues.redhat.com/browse/KEYCLOAK-2940 */
     @Path("logout")
+    @Deprecated
     public Object logout() {
```

Comment:
Looks like this logout endpoint has been deprecated for some time, do we have an issue to track it's removal?

Reply:
I am not 100% sure but I think the comment does not target the whole endpoint but just the functionality within the endpoint that is deprecated, see e.g. https://github.com/keycloak/keycloak/blob/a65508ca13da5ad41ae0f4aa045ae0dd6219f3dc/services/src/main/java/org/keycloak/protocol/oidc/endpoints/LogoutEndpoint.java#L165-L185

Reply:
You're right, I've removed the `@Deprecated` annotation and the comment.